wazo_webhookd/controller.py

```diff
@@ -80,12 +83,18 @@ def run(self):
             self._celery_process.terminate()
             logger.debug('waiting for remaining threads/subprocesses...')
             self._celery_process.join()
+            self._stopping_thread.join()
             logger.debug('all threads and subprocesses stopped.')
 
     def stop(self, reason):
         logger.warning('Stopping wazo-webhookd: %s', reason)
-        self.rest_api.stop()
+        self._stopping_thread = threading.Thread(target=self.rest_api.stop, name=reason)
+        self._stopping_thread.start()
 
 
 def _sigterm_handler(controller, signum, frame):
     controller.stop(reason='SIGTERM')
+
+
+def _sigint_handler(controller, signum, frame):
+    controller.stop(reason='SIGINT')
```

Comment:
If both functions have the exact same logic, can't we just have one handler, eg`signal_handler`?
You should be able to get the "reason" from the `signum` Enum with `signal.Signals(signum).name`.

Reply:
Oh you're right!
Done, thank you